src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -605,6 +605,10 @@ public function testChgrpSymlink()
      */
     public function testChgrpSymlinkFails()
     {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('Shouldn\'t be executed in HHVM context.');
```

Comment:
Correct me if i am wrong but this tests seems to be testing if i try to change a symlink file to a inexistent group, but as said above `lchgrp` returns `true` in HHVM. I don't know if this is a expected behaviour in HHVM, i can open an issue in facebook/hhvm to be sure.